prov/verbs/src/verbs_cq.c

```diff
@@ -36,6 +36,7 @@
 
 #include "fi_verbs.h"
 
+static uint64_t fi_ibv_comp_flags(struct ibv_wc *wc);
```

Comment:
Thank you, I'm going to move the function.
Who shall I set for the signed-off line in the commit messages for the verbs provider?